tests/tests.coffee

```diff
@@ -197,7 +197,7 @@ class VexTabTests
 
     # KEY SIGNATURE TESTS
 
-    for key of Vex.Flow.keySignature.keySpecs
+    for key of Vex.Flow.getKeySignatures()
```

Comment:
Similarly, I added a getter here instead of allowing client code to reach into the internals of VexFlow.